src/test/java/com/epam/repository/PersonDAOTest.java

```diff
@@ -39,7 +37,7 @@ public void setUp() {
     public void buildPersonPositiveCheck() throws SQLException {
         int testId = 1;
         when(resultSet.getInt("id")).thenReturn(testId);
-        when(resultSet.getString("nickname")).thenReturn(testUserName);
+        when(resultSet.getString("email")).thenReturn(testUserName);
```

Comment:
It is not about hotel...

Reply:
Our tests are crashing because of last push in dev, where we changed one of the field in table of Person entity.